llmfoundry/callbacks/eval_gauntlet_callback.py

```diff
@@ -66,13 +84,12 @@ def __init__(self,
             )
 
         self.categories = categories
+        self.category_names = [conf.get('name', '') for conf in self.categories]
```

Comment:
was trying to understand this line and realized the docstring doesn't mention categories. can you update please?